docker-compose.yml

```diff
@@ -76,7 +76,7 @@ services:
     # ZooKeeper
     #
     zookeeper-base:
-        image: ${ZOOKEEPER_IMAGE:-zookeeper:3.7}
```

Comment:
I was actually thinking about adding 3.8 in addition to 3.7, and update the scripts so that the test will be run on both to have one common table instead of multiple, and that way you can update README (which is the only file in this repository that has some chances that it will be seen by others) with new numbers.

Reply:
Understood, I'll then add separate containers for version 3.8.